content/blog/2018/02/2018-02-27-browser-usage.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Browser Market Share on Jenkins.io"
+tags:
+- community
+- stats
+- browser
+author: olblak
+---
+
+= Browser Market Share on Jenkins.io
+
+I recently had a look to the google analytics account used for the main jenkins website. +
+It was interesting to discover which browsers are used by Jenkins visitors and how this compares with other websites. +
+I decided to analyse one year of data from January 2017 to January 2018. +
+I selected http://gs.statcounter.com/about[statcounter] as an external data source in order to compare results from jenkins.io.
+
+
+== Jenkins Visitors
+
+Let's talk about numbers: +
+During that period, 3,496,245 users across the world visited this website. +
+This is a growth of 152.69% visitors compared to the year before. +
+92.28% of them used a desktop, 6.63% were on mobile and only 1.10% on a tablet.
+
+[cols="a,a"]
+|===
+|
+|
+|
+.Jenkins Visitors' Browser Usage
+[cols="1,10,1",width="100%"]
+!===
+!#! Browser! %
+
+! 1. 
+! Chrome
```

Comment:
Maybe just have it as a legend on the image?

Reply:
I am not sure to understand what you mean here, instead of a table with browser + pourcentage, I should put a legend under the image?